plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/DevWatchBootableJarMojo.java

```diff
@@ -187,9 +204,33 @@ public final class DevWatchBootableJarMojo extends AbstractDevBootableJarMojo {
     @Parameter(alias= "debug-suspend", defaultValue = "false", property = "wildfly.bootable.debug.suspend")
     private boolean debugSuspend;
 
+    /**
+     * Enable/Disable remote connection.
+     */
+    @Parameter(defaultValue = "false", property = "wildfly.bootable.remote")
+    private boolean remote;
+
+    /**
+     * Remote connection username.
+     */
+    @Parameter(property = "wildfly.bootable.remote.username")
```

Comment:
org.wildfly.plugin.common.PropertyNames contains all the properties, could be a good idea to add the new one there too.